docs/_documentations/troubleshooting.md

```diff
@@ -75,6 +75,27 @@ When using OS authentication setups (for example, AzureAD), Docker Shared Drive
 2. Create the new account with the same username but without the prefix (for example, if your AzureAD account is `AzureAD/BobSmith`, your new local account should be `BobSmith`). Use the same password as your other account.
 3. Select your new local account and click **Change account type**. Select the dropdown menu and select **Administrator**. Share the drive again in Docker.
 
+<!--
+Action/Topic: Installing Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind/issues/436
+0.6.0: Still present
+-->
+## Error appears after installing or updating Codewind
+After you install or update Codewind, an error might appear when you try to start Codewind.
+
+**Workaround:** Stop Codewind. Understand that the `docker system prune -a` command removes the following items:
```

Comment:
> Stop Codewind. 
Remove the Codewind docker images. In VS Code, use the Remove Codewind Images command. In Eclipse, use the Uninstall action.
If this fails for any reason, the images can also be removed through the Docker command line using `docker image ls` to identify the Codewind images, followed by `docker image rm` to remove them.

Users should try above first, after stopping. If that still fails, then they can try the system prune.

Reply:
Great! I will add this in a new commit!